catalog/spatial/csw/spatial-csw-source/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -110,7 +110,7 @@
     </OCD>
 
     <OCD name="CSW Federation Profile Source"
-         description="DDF's full fidelity CSW Federation Profile. Use this when federating to a DDF based system."
+         description="The full fidelity CSW Federation Profile. Use this when federating to a DDF based system."
```

Comment:
✏️  We may want to remove the `The` here to be consistent with the other descriptions.